rudder-agent/SOURCES/debug-script/tests/rudder_runtime

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# runtime test
+#
+
+# Describe your tests here
+runtime_test_description="Verify if some runs took more than amount of time between runs"
+  
+runtime_test() {
+
+  # By default, this test is successful
+  RUNTIME_AGENT='ok'
+  
+  Path_LOG="/var/rudder/cfengine-community/outputs"
+  runtime=300
+    if [ -d "$Path_LOG" ] 
+    then
+      # Get a last date access 
+      date_access=$(stat -c %X "$Path_LOG")
+      
+      # Get a last date of mofification
+      date_modif=$(stat -c %Y "$Path_LOG")
+      
+      # Get a time of runtime_agent
+      runtime_agent=$(($date_modif-$date_access))
```

Comment:
How can the difference between last write (by agent) and last read (by any user) represent the runtime ?

Please use the file name to extract the start date.